src/main/resources/static/js/MainWindow.js

```diff
@@ -852,7 +852,7 @@ function importData() {
 							return false;
 						}
 						parsed = parseInt(currentInput, 10);
-						if (!(currentInput === parsed) || parsed < 0) {
+						if (!(currentInput == parsed) || parsed < 0) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4872216)